pkg/trace/sampler/dynamic_config_test.go

```diff
@@ -161,7 +161,7 @@ func BenchmarkRateByService(b *testing.B) {
 		{"three", "test"}:  0.33,
 		{"one", "prod"}:    0.12,
 		{"five", "test"}:   0.8,
-		{"six", "staging"}: 0.9,
+		{"rtloader", "staging"}: 0.9,
```

Comment:
maybe we should limit your script to `cmd`, `six`, `tasks` and `pkg/collector` folder ?